pkgs/applications/misc/foot/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, fcft, freetype, pixman, libxkbcommon, fontconfig, wayland
+, meson, ninja, ncurses, scdoc, tllist, wayland-protocols, pkg-config
+}:
```

Comment:
```suggestion
{ stdenv
, fetchgit
, pkg-config
, fcft
, freetype
, pixman
, libxkbcommon
, fontconfig
, wayland
, meson
, ninja
, ncurses
, scdoc
, tllist
, wayland-protocols
}:
```

Reply:
For my taste these just convert horizontal space into vertical space without a real benefit to readability. I reduced horizontal space somewhat and split the inputs by category more obviously.